.github/workflows/test_iree.yml

```diff
@@ -101,11 +101,15 @@ jobs:
           - name: cpu_llvm_sync
             runs-on: nodai-amdgpu-w7900-x86-64
             models-config-file: config_pytorch_models_cpu_llvm_task.json
-            sdxl-config-file: config_sdxl_scheduled_unet_cpu_llvm_task.json
+            sdxl-prompt-encoder-config-file: config_sdxl_prompt_encoder_cpu_llvm_task.json
+            sdxl-unet-config-file: config_sdxl_scheduled_unet_cpu_llvm_task.json
+            sdxl-vae-decode-config-file: config_sdxl_vae_decode_cpu_llvm_task.json
```

Comment:
Fine for now, but let's figure out how to move these from the current `.json` config files to some other format (e.g. `.py` files or another `.json` format). I don't like special casing SDXL and this goes even further to special case _each submodel_.

Reply:
Ok that sounds good